.travis.yml

```diff
@@ -4,7 +4,7 @@ language: ruby
 branches:
   only: [master]
 rvm:
-  - 2.2.4
+  - 2.2.10
```

Comment:
If you don't include the patch version, Travis will use one of the pre-compiled rubies from that trunk. 
Faster and your specs will keep getting tested with the latest patch available on Travis all the time.
```
rvm:
 - 2.2
```